pkg/config/kv/kv_node_test.go

```diff
@@ -70,8 +70,8 @@ func TestDecodeToNode(t *testing.T) {
 		{
 			desc: "several entries, level 0",
 			in: map[string]string{
-				"traefik": "bar",
-				"traefic": "bur",
+				"traefik":  "bar",
+				"traefik_": "bur",
```

Comment:
I tend to suggest using a `_` instead of misspelling something when one needs a variation on an item, but one could also use a numeric suffix (e.g. `1` or `2`)